lib/basedriver/helpers.js

```diff
@@ -321,7 +321,12 @@ async function unzipApp (zipPath, dstRoot, supportedAppExtensions) {
   const tmpRoot = await tempDir.openDir();
   try {
     logger.debug(`Unzipping '${zipPath}'`);
-    await zip.extractAllTo(zipPath, tmpRoot);
+    // zip.extractAllTo could not unzip Unicode filenames properly by Xcode
```

Comment:
I would rather replace the implementation of `extractAllTo` inside appium-support

Reply:
Yeah, I searched which repositories used `extractAllTo`. Will create a PR to the support after checking

Reply:
I found the unzipper did not keep permission. In chromedriver download feature, unzipper did unzip the binary as 644. It should have been 755. So, I think this unzipper should be used as a spot like this case rather than replacement of general extractAllTo.

Reply:
Could you please describe it as a bug to unzipper repository?: https://github.com/ZJONSSON/node-unzipper/issues

Are you sure that extracted files will be working as expected on real/virtual devices after being extracted without original permissions preserved?

Reply:
https://github.com/EvanOxfeld/node-unzip/issues/123 and https://github.com/ZJONSSON/node-unzipper/issues/131 are about the permission/mode stuff.

> Are you sure that extracted files...

Yes, only our test apps and a simple app I created to check the Unicode issue though.

But I think we should keep all files as is, so should close this PR.